tests/PHPUnit/PredisCommandTestCase.php

```diff
@@ -51,10 +51,10 @@ public function getCommand(): Command\CommandInterface
      * Returns a new client instance.
      *
      * @param bool $flushdb Flush selected database before returning the client
-     *
+     * @param array|null $parameters
      * @return Client
      */
-    public function getClient(bool $flushdb = true): Client
+    public function getClient(bool $flushdb = true, ?array $parameters = null): Client
```

Comment:
I quite dislike this addition, can you call `createClient()` directly, since you're not using the `$flushdb` argument anyhow?

Reply:
Makes sense, didn't notice that `createClient()` is `protected` not a `private`